src/shared/components/annotation/oncokb/RefComponent.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {getNCBIlink} from "../../../api/urls";
+import mainStyles from "../styles/oncokb/main.module.scss";
+import ReferenceList from "./ReferenceList";
+import DefaultTooltip from "../../defaultTooltip/DefaultTooltip";
+import {ICache} from "../../../lib/SimpleCache";
+
+export default class RefComponent extends React.Component<{ str: string, componentType: 'tooltip' | 'linkout', pmidData:ICache<any>}> {
```

Comment:
`str` doesn't sound very informative as a property name, can we call this something else?

Reply:
Changed to `content`